database_creation.py

```diff
@@ -28,4 +28,4 @@ def connect_to_db():
 def show_databases():
     cur.execute("SHOW DATABASES")
     for database in cur:
-        print(database)
+        print(database)
```

Comment:
Add a newline at the end of the file.

Reply:
A newline at the end of the file was added.